src/configuration/GPG/GnuPG/params.txt

```diff
@@ -0,0 +1,9 @@
+Key-Type: RSA
+Key-Length: 8192  
```

Comment:
Elliptic curves are still experimental and not yet usable, see below. But yes, in general I totally agree with you, @schallertd.

@azet I think the key lifetime (expiration date) has nothing to do with the key size. A message I wrote to someone in the past is today still encrypted with the public key that was known at this time, regardless of the keys lifetime.
If Mallory intercepted the 2048 bit encrypted message a year ago, who knows maybe some instance is able to break 2048 RSA in the near future. The point is we do not know this, we only know for sure that some instances put a lot of money and brainpower in this matter.

So in my conclusion, if RSA has not a mathematical or [implementation flaw](https://eprint.iacr.org/2017/627.pdf) the only real parameter one can address to keep messages as long as possible confident is the key size. An RSA 8091 bit key is the largest possible and its usable in practice.